backend/store.py

```
@@ -1352,11 +1442,46 @@ def get_job(self, job_id: UUID) -> Optional[AgentJob]:
+
+    def append_job_log(self, job_id: UUID, message: str) -> None:
+        key = self._job_logs_key(job_id)
+        # Store chunks (may contain multiple lines).
+        self.client.rpush(key, message)
+        ttl_seconds = int(os.getenv("JOB_LOG_TTL_SECONDS", "604800"))  # 7 days
+        try:
+            if ttl_seconds > 0:
+                self.client.expire(key, ttl_seconds)
+        except Exception:
+            # Best-effort TTL: not all clients/backends support expire the same way.
+            pass
```

Ruff (0.14.8)

`try`-`except`-`pass` detected, consider logging the exception

(S110)